google-datacatalog-kafka-connector/tests/google/datacatalog_connectors/kafka/test_utils/test_utils.py

```diff
@@ -14,16 +12,26 @@ def make_entry_for_topic(self, topic):
         return entry_id, entry
 
 
-class FakeMetadataScraper(MetadataScraper):
+class FakeKafkaConsumer(mock.MagicMock):
 
-    def _get_metadata_from_message_broker_connection(self, connection_args):
-        return mock.MagicMock()
+    def list_topics(self, timeout=-1):
+        raw_metadata = mock.MagicMock()
```

Comment:
so is there any reason to use MagicMock instead of some hardcoded dicts here?

Reply:
to convey the idea that original list_topics() returns a python object of a ClusterMetadata class, that is potentially more than a dict. At the same time, from functionality perspective, we can just do hardcoded dict. Do you find it will be easier to read that way?

Reply:
in general I'm against using mocks if they aren't needed. If the 'real' version of this object returns a ClusterMetadata object, can we do the same here (perhaps passing the hardcoded values to the ClusterMetadata constructor?

Reply:
Hm, it is possible, I think, just not through the constructor, but by setting the instance variables, as [source suggests](https://docs.confluent.io/current/clients/confluent-kafka-python/_modules/confluent_kafka/admin.html#ClusterMetadata). I'll try to refactor it then.

Reply:
Done